example/ios/Podfile.lock

```diff
@@ -296,7 +296,7 @@ PODS:
     - React-Core (= 0.63.2)
     - React-cxxreact (= 0.63.2)
     - React-jsi (= 0.63.2)
-  - RNDateTimePicker (2.6.1):
+  - RNDateTimePicker (3.0.2):
```

Comment:
don't you want to have it set to the latest version (3.4.7)?

Reply:
This is just for the example app (which from the lock is depending on 3.0.2) 👍